common/src/main/java/com/microsoft/identity/common/adal/internal/AuthenticationConstants.java

```diff
@@ -827,6 +837,32 @@ private AuthenticationConstants() {
     }
 
     /**
+     * Represents the constants value for Microsoft STS.
+     */
+    public static final class MSSTS {
```

Comment:
I think these don't need to be in a separate constants class.  I think they can be held closer to the actual implementation within common.